test/check-sosreport

```diff
@@ -25,7 +25,6 @@ import subprocess
 import unittest
 
 @unittest.skipIf("debian" in os.environ.get("TEST_OS", ""), "No sosreport on Debian.")
```

Comment:
My debian 8 (jessie) has the package sosreport. Is that outdated or not compatible with what we do? Do we just need to build a new debian image?